torch/_dynamo/variables/user_defined.py

```diff
@@ -442,6 +446,17 @@ def call_function(
             else:
                 var_cls = GenericContextWrappingVariable
 
+            if self.value is contextlib._GeneratorContextManager and isinstance(
```

Comment:
This change is potentially risky, can we add a config flag for it so that it's easy to turn off?

Reply:
The next PR you mean https://github.com/pytorch/pytorch/pull/138203?